### ISSUE TITLE
Issue timeout-or-duplicate

### ISSUE THREAD

**Issue by Nuranto** [RyanDaDeng/laravel-google-recaptcha-v3#31]
I often (but not always..) get `timeout-or-duplicate` on first submit try, and on second submit the POST pass.
It seems it can happen when the validation process is run twice..? Any idea ?

(Also, even if it fails, isn't it suppose to fallback on V2 ? I thought it was suppose to, but i don't see it anymore in README, maybe I mix up with some other project ?)

**Comment by RyanDaDeng** [comment]
Hello @Nuranto , I can answer "isn't it suppose to fallback on V2" this question first, the v3 won't fallback to v2 actually, v3 is only collecting score from your site action, and you determine your own score threshold. If the score less than threshold, you need to do your own fallback, the official site suggests to use v3 with two-factor authentication.

Yeah, in previous readme, this is included in TODO list, I removed it because it requires to load google script twice which doesn't make sense and slow the site actually....

For your timeout-duplicate problem, can you briefly share your file arrangement? like how you manage to display the recaptcha v3 in blade or vue.

**Comment by Nuranto** [comment]
> the official site suggests to use v3 with two-factor authentication.

In my case, I'm not using it for auth, but for contact and comment forms.  But I see the idea.

> Yeah, in previous readme, this is included in TODO list, I removed it because it requires to load google script twice which doesn't make sense and slow the site actually....

Isn't it possible to load V2 script only in case of failure ?


> For your timeout-duplicate problem, can you briefly share your file arrangement? like how you manage to display the recaptcha v3 in blade or vue.

``` 
<form id="contact-form" method="post" action="{{ route('contact.submit') }}" role="form">
        @csrf
[....]
                    <div id="frontcontact" class="float-right"></div>
                    {!!  GoogleReCaptchaV3::render(['frontcontact'=>'frontcontact']) !!}
                    <input type="submit" class="btn btn-send btn-primary" value="Send message">
[....]
</form>
```

And validation array item : 
```
'g-recaptcha-response' => [new GoogleReCaptchaV3ValidationRule('frontcontact')]
```

**Comment by RyanDaDeng** [comment]
Hello @Nuranto ,

sorry for the delay.

Here is my suggestions:

1. Ensure your version is higher or equal to 2.1.11
2. Ensure that   {!!  GoogleReCaptchaV3::init() !!} always located at the bottom no matter page you load.
3. Use {!! GoogleReCaptchaV3::renderField('frontcontact','frontcontact') !!} instead

If still not work, i will dig it into deep.

**Comment by Nuranto** [comment]
Hello @RyanDaDeng,

I looked into it, and it seems there was some conflict with my config files. I spare you the details, but I have some specifities with config file. Anyway, I found out that the app was using a config file where googlerecptchav3.setting was empty. 
So I guess, the only validation check was if the `g-recaptcha-response` field was filled or not (which seemed enough for stopping spam...), but caused `timeout-or-duplicate`.

So sorry, my bad on this one ! It works fine now.

Thank you for your answer and your time !
